tests/test_fs.py

```diff
@@ -43,12 +51,12 @@ def test_ls(fs: V3ioFS, new_file, path_cls):
 
 
 def test_ls_with_marker(fs: V3ioFS, new_file):
-    for i in range(1200):
+    for i in range(5):
```

Comment:
Why do you want to avoid from testing it on large amount of files?

Reply:
Added a limit of 3 to force this scenario without turning the test into a benchmark.